### ISSUE TITLE
Bonestab color is supposed to be fully red.

### ISSUE THREAD

**Issue by iKNIFEu** [Jcw87/c2-sans-fight#88]
Why hasn't anyone EVEN noticed this? Bonestab warning is not supposed to be THAT color. you play the game yourself! The color is supposed to be fully red.

**Comment by Jcw87** [comment]
When it came to adding this border, I couldn't just grab an art asset from the game, as it was likely just a box drawing function being called from the gamemaker code. At the time I added this asset, I hadn't obtained a working gamemaker decompiler yet, so my best source for picking the color came from the videos I recorded. If you have ever zoomed in on a video before, you might notice something that would make it difficult to obtain the correct color.

![image](https://user-images.githubusercontent.com/975426/104143456-59a17f80-5374-11eb-85d8-0900e6652cd2.png)

This doesn't happen with PNG screenshots, but I wasn't particularly enthusiastic about booting up the game just to get 1 screenshot of an attack that occurs late into the fight. That's why I recorded videos in the first place.

I will be investigating this.
